File: API/Entities/Relic.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using API.Extensions;

namespace API.Entities
{
    [Table("Relics")]
    public class Relic
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string NameUnmark { get; set ; }
        public string Title { get; set; }
        public string Content { get; set; }
        public int View { get; set; }
        public AppUser AppUser { get; set; }
        public int AppUserId { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;
        public ICollection<Photo> Photos { get; set; }
    }
}

File: client/src/app/_models/relic.ts
import { Photo } from "./photo";


export interface Relic {
    id: number;
    name: string;
    title: string;
    content: string;
    view: number;
    photoUrl: string;
    isApproved: boolean;
    isReject: boolean;
    created: Date;
    author: string;
    photos: Photo[];
}
